service/src/test/java/greencity/service/HabitAssignServiceImplTest.java

```diff
@@ -287,12 +289,36 @@ void unenrollHabitThrowWrongIdException() {
     }
 
     @Test
-    void deleteHabitAssign() {
+    void changeStatusHabitAssign() {
         HabitAssign habitAssign = ModelUtils.getHabitAssign();
-        when(habitAssignRepo.findByUserIdAndHabitId(1L, 1L)).thenReturn(Optional.ofNullable(habitAssign));
-        assert habitAssign != null;
-        habitAssignService.deleteHabitAssign(1L, 1L);
-        verify(habitAssignRepo).delete(habitAssign);
+        habitAssign.getHabit().setHabitAssigns(generateHabitAssignList());
+        when(habitAssignRepo.findListByUserIdAndHabitIdAndStatusIsInProgress(1L, 1L))
+            .thenReturn(habitAssign.getHabit().getHabitAssigns());
+        habitAssignService.cancelAllHabitAssign(1L, 1L);
```

Comment:
please use different ids for `userId` and `habitId` to be sure that everything works fine.
If someone evidently swaps `userId` and `habitId` this test will still work because you are using the same ids for `userId` and `habitId`
```
public void cancelAllHabitAssign(Long habitId, Long userId) {
        List<HabitAssign> habitAssigns = habitAssignRepo.findListByUserIdAndHabitIdAndStatusIsInProgress(userId, habitId);
        ...
    }
```
we can easily to check this case if we use different `ids`